### PR TITLE
Fix comments pointing to future points.

### PR DESCRIPTION
**Pull request by ryanwilson**

### CODE REVIEW

File: tutorials/run-swift-kitura-on-google-cloud/index.md [hunk 1]
```diff
@@ -73,10 +73,10 @@ We'll use the [Swift Package Manager][spm] to manage our app's dependencies.
         let router = Router()
 
         // Respond to App Engine health check requests
-        ...
+        ... see #2
```
Comment (jmdobry) [inline tutorials/run-swift-kitura-on-google-cloud/index.md]: Change from:

```
... see #2
```

to

```
// TODO: see #2
```

File: tutorials/run-swift-kitura-on-google-cloud/index.md [hunk 2]
```diff
@@ -73,10 +73,10 @@ We'll use the [Swift Package Manager][spm] to manage our app's dependencies.
...
 
         // Basic GET request
-        ...
+        ... see #3
```
Comment (jmdobry) [inline tutorials/run-swift-kitura-on-google-cloud/index.md]: Change from:

```
... see #3
```

to

```
// TODO: see #3
```

File: tutorials/run-swift-kitura-on-google-cloud/index.md (hunk 1 above)
Comment (ryanwilson) [inline tutorials/run-swift-kitura-on-google-cloud/index.md]: Done.

File: tutorials/run-swift-kitura-on-google-cloud/index.md (hunk 2 above)
Comment (ryanwilson) [inline tutorials/run-swift-kitura-on-google-cloud/index.md]: Done.

**Comment by jmdobry** [comment]
Thanks!

**Pull request merged**